test/parallel/test-tls-env-extra-ca-no-crypto.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+// Flags: --expose-internals
+
+const common = require('../common');
+const fixtures = require('../common/fixtures');
+const assert = require('assert');
+const { fork } = require('child_process');
+
+// This test ensures that trying to load extra certs won't throw when
+// there is no crypto.
```

Comment:
Nit: Maybe we can just run this test only when crypto is not there, just to emphasize?

Reply:
It seems that we don't run tests with `--without-ssl` build. I would like and change the comment a bit and keep the test.